### PR TITLE
Changing averageRating type to numeric

### PR DESCRIPTION
**Pull request by danghieutrung**

### CODE REVIEW

File: src/postgres/format_tables.sql [hunk 1]
```diff
@@ -3,6 +3,8 @@ ALTER TABLE titles
     USING
         string_to_array(genres, ',')::varchar(20)[];
 
+ALTER TABLE ratings
+    ALTER COLUMN averageRating TYPE numeric(2,1);
```
Comment (kiendang) [inline src/postgres/format_tables.sql]: We should just change the type in `create_tables.sql`. No point in loading it in `double precision` first then changing the type.

**Review by kiendang — CHANGES_REQUESTED** [review]

File: src/postgres/format_tables.sql (hunk 1 above)
Comment (danghieutrung) [reply src/postgres/format_tables.sql]: I just pushed a new commit as per your request to change the type in `create_tables.sql` file instead of `format_tables.sql`

File: src/postgres/create_tables.sql [hunk 2]
```diff
@@ -38,7 +38,7 @@ CREATE TABLE episodes (
 DROP TABLE IF EXISTS ratings CASCADE;
 CREATE TABLE ratings (
     tconst char(10) primary key,
-    averageRating double precision,
+    averageRating numeric(2,1),
```
Comment (kiendang) [inline src/postgres/create_tables.sql]: Got a `numeric field overflow` error `A field with precision 2, scale 1 must round to an absolute value less than 10^1.` so apparently there are ratings that are more precise than this. Just change this to `real` (the smallest postgres variable-precision numeric type) then?

File: src/postgres/format_tables.sql [hunk 3]
```diff
@@ -3,7 +3,6 @@ ALTER TABLE titles
     USING
         string_to_array(genres, ',')::varchar(20)[];
 
-
```
Comment (kiendang) [inline src/postgres/format_tables.sql]: NIT: undo this change.

**Review by kiendang — CHANGES_REQUESTED** [review]

File: src/postgres/create_tables.sql (hunk 2 above)
Comment (danghieutrung) [reply src/postgres/create_tables.sql]: The error must happen in the case of `averageRating` equal to 10.0, which has 3 digits and requires a precision of 3. Replacing with `numeric(3,1)` should fix the error but setting the type to `real` is also a fine solution.

Comment (kiendang) [reply src/postgres/create_tables.sql]: `numeric(2)` instead of `numeric(2, 1)` would work in this case. However we should set `real` here regardless, coz this is external data we cannot control. Even if we want something like `numeric(2)` we should set `real` here and add an `alter table` clause in `format_tables.sql` to cast the column to `numeric(2)` (this part is optional for this PR).
